docs/patterns/Auth/connected_account.mdx

```diff
@@ -1,260 +1,167 @@
+- If you have two users, **Jessica** and **Melissa**, they will each have **unique entity IDs**.
+<Info>If not specified, `default` will be used as the entity ID.</Info>
 
-### Code Snippets for Connecting an Account
-<Steps>
-<Step title="Collect Parameters Required to create a Connection">
-To create a connected account for your user:
-
-1. Fetch required auth parameters for the application
-2. Collect these parameters from the user
-3. Initiate a new connection request
-
-<Note> Linear: Requires `api_key`, Shopify: Requires `shop_name` for OAuth flow </Note>
+You can use an entity object to manage connected accounts and perform actions on behalf of a user (more on this [here](/patterns/tools/use-tools/action-guide-without-agents#how-can-i-use-tools-for-a-specific-user)). Here's how to retrieve an entity object:
 
 <CodeGroup>
-```python Python - Fetch Auth Parameters
-from composio import ComposioToolSet, App
-toolset = ComposioToolSet()
+```python Python
+from composio import ComposioToolSet
 
-response = toolset.get_expected_params_for_user(app=App.LINEAR) # can use integration_id instead of app
-
-print(response["expected_params"])
+toolset = ComposioToolSet()
+entity = toolset.get_entity(id='Jessica')
 ```
 
-```javascript Javascript - Fetch Auth Parameters
-import { Composio, OpenAIToolSet } from "composio-core";
+```javascript JavaScript
+import { OpenAIToolSet } from "composio-core";
 
 const toolset = new OpenAIToolSet();
-
-const expectedInputFields = await toolset.getExpectedParamsForUser({
-    app: "gmail",
-});
-
-console.log(expectedInputFields)
+const entity = toolset.client.getEntity(id='Jessica');
 ```
 </CodeGroup>
 
-Collect the required parameters from your user through your preferred interface (UI, chat, or API).
-
-<Note>
-All parameters should be provided as strings.
-</Note>
-
-</Step>
-
-<Step title="Try creating a connection with the parameters collected">
+### Connected Accounts
+- When a user connects their account, a `connected_account` object is created. 
+- **Connected Account** securely stores **authentication data** such as **API keys, access tokens and refresh tokens**.
 
-In the code snippets below replace `redirect_url`
+### How to initiate a new connection for your user
+Follow these steps to initiate a new connection for your user (for both OAuth and non-OAuth based Apps):
 
+<Tabs>
+<Tab title="OAuth-Based Apps">
+Initiate a new connection by passing the redirect url, entity id & app.
 <CodeGroup>
-```python Python - New Connection
-
-# This is the URL that the user will be redirected to after completing the authentication process
-redirect_url = "https://yourwebsite.com/connection/callback"
+```python Python
+from composio import ComposioToolSet, App 
+toolset = ComposioToolSet()
 
-entity_id = "Jessica" # This is the unique identifier for the user
+# URL that the user will be redirected to after completing the authentication process
+redirect_url = "https://yourwebsite.com/connection/success"
```

Comment:
The redirect URL pattern has been standardized to `https://yourwebsite.com/connection/success` across documentation, but there should be a clear note in the documentation that this is just an example URL and should be replaced with the actual application's callback URL. This helps prevent confusion for new users who might copy-paste the example directly.